src/index.ts

```diff
@@ -31,6 +34,21 @@ async function assertFileExists(path: string) {
   );
 }
 
+function processBundleBudgets() {
+  // bundles inputs will be dynamic
+  // e.g bundle-{name}: 50
+  const bundleBudgets: BundleBudget[] = [];
+  for (let [k, v] of Object.entries(process.env)) {
+    // all github action inputs have this prefix "INPUT_", the real input starts with bundle-
```

Comment:
**curious** Do they? Is this because you're parsing `process.env` and not using `core.getInput`?

Is there a way we could rely on `core.getInput` instead? That would remove the need to parse user input this way.